docs/ufs.md

```diff
@@ -0,0 +1,49 @@
+## UFS - universal file system
+
+Since ESP8266-12F has 4MB flash you can build a variant with a `filesystem` and store your files there (data, images, commands, etc).<br>
+The file partition does NOT get erased by reset commands or factory default. Only a flash erase will wipe it.<br>
+There are some special files that you can upload and use:<br>
+
+**autoexec.bat**<br>
+Stores commands that will be executed at every boot, similar to the backlog commands in rules trigger at `System#Boot`.<br>
+Almost any commands can be used.<br>
+However, avoid commands that will make the device to reboot (examples: changing Wifi setting, MQTT settings, Templates & GPIO, etc)<br>
+otherwise it will reboot continuously (in fact 3 or 4 times then autoexec will be disabled after boot loop detection)<br>
+Commands must be kept to one command per line; they will be executed sequentially.<br>
+
+**display.ini**<br>
+Stores data that will be displayed at every boot, similar to the DisplayText commands in rules trigger at `System#Init` (as long as you have a display driver initializated).<br>
+example:<br>
+```; clr screen
```

Comment:
The triple-ticks should be alone on the line and `; clr screen` should be on the next line